#! /bin/sh /usr/share/dpatch/dpatch-run
## 17-init-setup.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove setup option from initscript

@DPATCH@

--- virtualbox-ose-2.2.2-dfsg/src/VBox/Installer/linux/vboxdrv.sh.in.orig	2009-05-04 16:47:42.000000000 +0200
+++ virtualbox-ose-2.2.2-dfsg/src/VBox/Installer/linux/vboxdrv.sh.in	2009-05-04 16:47:58.000000000 +0200
@@ -339,9 +339,6 @@
     stop
     start
     ;;
-setup)
-    setup
-    ;;
 status)
     dmnstatus
     ;;
